.github/workflows/e2e-test.yml

```diff
@@ -61,6 +61,8 @@ jobs:
           arch : x86_64
           emulator-options: ${{ vars.EMULATOR_OPTIONS }}
           script: bundle exec fastlane run_e2e_test batch:${{ matrix.batch }} batch_count:${{ strategy.job-total }}
+        env:
+          STREAM_DEMO_APP_SECRET: ${{ secrets.STREAM_DEMO_APP_SECRET }}
```

Comment:
What is this secret used for?
Do we have it stored somewhere else? (eg: 1Password)

Reply:
Added a comment regarding this at the top of the `AuthTests.kt` 👍